packages/replay-internal/src/coreHandlers/util/fetchUtils.ts

```diff
@@ -192,7 +193,8 @@ function getResponseData(
 
     return buildNetworkRequestOrResponse(headers, size, undefined);
   } catch (error) {
-    DEBUG_BUILD && logger.warn('[Replay] Failed to serialize response body', error);
+    DEBUG_BUILD && logger.warn('Failed to serialize response body');
```

Comment:
I think the one improvement we can do here more is to avoid having to double-call this in so many places. Instead, what about we make this something like this:

```js
DEBUG_BUILD && logger.exception(error, 'Failed to serialize response body');
```

where the second argument is optional, and if defined will just `logger.error` this string? This may safe a few more bytes, and IMHO is also nicer from a usage experience...?

Reply:
Updated, though it didn't seem to help bundle size too much